packages/svelte-ux/src/lib/components/SelectField.svelte

```diff
@@ -352,10 +374,15 @@
     on:blur={onBlur}
     on:keydown={onKeyDown}
     on:keypress={onKeyPress}
-    actions={(node) => [selectOnFocus(node)]}
-    class={cls('h-full', theme.field, classes.field)}
+    actions={fieldActions}
+    classes={{
+      root: cls('h-full'),
+      ...theme.field,
+      ...fieldClasses,
```

Comment:
since these are all objects, if anyone defines something (like `root`), it will remove all the other classes.  This might be OK, just thinking it through...

Reply:
Oh, true, it wouldn't be additive in that case. Not sure how much I like that...if only you could easily pipe each overlapping property through `cls()`...hmmm.

Reply:
Could maybe use [merge()](https://github.com/techniq/svelte-ux/blob/main/packages/svelte-ux/src/lib/utils/object.ts#L66-L79), although it uses lodash, and I've been trying to ween the project off of it (either have it's own impl in the project, or use [just](https://github.com/angus-c/just/), although I don't think it had a similar merge as lodash's `mergeWith`

Reply:
Reverted this change.